tests/baselines/reference/instantiationHandledLargeVolumeOfSimpleWork.symbols

```diff
@@ -0,0 +1,207 @@

```

Comment:
🥳

Reply:
In my case, a MySQL `SELECT` statement can select more than 50 columns.

And they're concatenated to previous columns in the `SELECT` clause. So, being able to concat long tuples is super helpful.